### PR TITLE
Add start files and authentication

### PR DESCRIPTION
**Pull request by Ehsanmh77**

### CODE REVIEW

File: public/assets/fonts/shabnam/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,281 @@
+تغییرات فونت شبنم
```
Comment (pmzi) [inline public/assets/fonts/shabnam/CHANGELOG.md]: Is this file necessary? It is better to remove it.

File: public/assets/fonts/shabnam/LICENSE [hunk 2]
```diff
@@ -0,0 +1,94 @@
+Copyright (c) 2015, Saber Rastikerdar (saber.rastikerdar@gmail.com),
```
Comment (pmzi) [inline public/assets/fonts/shabnam/LICENSE]: Is this file necessary too?

Comment (pmzi) [reply public/assets/fonts/shabnam/LICENSE]: It is better to remove all font-related mds and texts

File: src/stylesheets/main.scss [hunk 3]
```diff
@@ -0,0 +1,6 @@
+@import '../../node_modules/normalize-scss/sass/normalize' ;
```
Comment (pmzi) [inline src/stylesheets/main.scss]: Create a file called `imports.scss` which includes abstracts, mixins, and functions of the scss

**Comment by pmzi** [comment]
Also we have folders with only having an `index.vue` inside them. This is incorrect and should be: `FolderName.vue` instead of `FolderName > index.vue`

**Comment by pmzi** [comment]
Also `scss`s should go inside `vue` files.

File: public/assets/fonts/shabnam/CHANGELOG.md (hunk 1 above)
Comment (Ehsanmh77) [reply public/assets/fonts/shabnam/CHANGELOG.md]: Yeah it's not 👍
